app/helpers/date-from-now.ts

```diff
@@ -1,21 +1,41 @@
-import { helper } from '@ember/component/helper';
 import { formatDistanceStrictWithOptions } from 'date-fns/fp';
+import Helper from '@ember/component/helper';
 
-const dateFromNow = helper(function dateFromNow(
-  [date]: [(Date | null | undefined)?],
-  { currentDate = new Date() }: { currentDate?: Date | undefined } = {},
-): string {
-  if (date === undefined || date === null) {
-    return '';
+interface DateFromNowSignature {
+  Args: {
+    Positional: Array<Date | null | undefined>;
+  }
+}
+
+export default class DateFromNow extends Helper<DateFromNowSignature> {
+  timer: number | null = null;
+
+  clearTimer() {
+    if (this.timer !== null) {
+      window.clearTimeout(this.timer);
+      this.timer = null;
+    }
   }
 
-  return formatDistanceStrictWithOptions({ addSuffix: true }, currentDate, date);
-});
+  compute(positional: [Date]) {
+    let [date] = positional;
+    this.scheduleRecompute();
+    return formatDistanceStrictWithOptions({ addSuffix: true }, new Date(), date);
+  }
+
+  scheduleRecompute() {
+    this.clearTimer();
+    this.timer = window.setTimeout(() => this.recompute(), 1000);
```

Comment:
@libmartinito one issue here is that different helpers will end up updating at different times - can we try to keep only one "timer" loop across all helpers? 

From linear: 

> Let's update this once every second, and let's make sure all timestamps across the app update at the same point of time so it isn't jarring.